superset/assets/src/dashboard/components/gridComponents/new/DraggableNewComponent.jsx

```diff
@@ -34,7 +36,7 @@ export default class DraggableNewComponent extends React.PureComponent {
         {({ dragSourceRef }) => (
           <div ref={dragSourceRef} className="new-component">
             <div className={cx('new-component-placeholder', className)} />
-            {label}
+            {t(label)}
```

Comment:
Pretty sure you want to use `t()` where the text is defined, not on the variable